util/reggen/gen_dv.py

```diff
@@ -41,11 +65,14 @@ def miname(m: Window) -> str:
 
 def gen_core_file(outdir: str,
                   lblock: str,
-                  dv_base_prefix: str,
+                  dv_base_names: str,
                   paths: List[str]) -> None:
     depends = ["lowrisc:dv:dv_base_reg"]
-    if dv_base_prefix and dv_base_prefix != "dv_base":
-        depends.append("lowrisc:dv:{}_reg".format(dv_base_prefix))
+    blocks_base_names = get_dv_base_names_objects(dv_base_names)
+    for block in blocks_base_names:
+        pkg_name = blocks_base_names[block].pkg
+        pkg_name = pkg_name.replace("_pkg", "")
+        depends.append("lowrisc:dv:{}".format(pkg_name))
```

Comment:
Directly inline here?
```suggestion
        depends.append(f"lowrisc:dv:{pkg_name[:-4]}")
```
Need to document somewhere that this fusesoc core naming convention is enforced. There are examples where the fusesoc core name has the `_pkg` suffix.

Reply:
Oh that is a good reminder, thanks and updated in the README!